client/src/components/Sidebar/ChatContent.js

```diff
@@ -36,6 +43,9 @@ const ChatContent = (props) => {
           {latestMessageText}
         </Typography>
       </Box>
+      {unreadMessagesCount !==0 && <Typography className={classes.unreadCounter}>
```

Comment:
Why does `UnreadCounter` need to be wrapped inside `Typography`?

Reply:
You are right. No need, removing this